test/common/tmpdir.js

```diff
@@ -5,17 +5,27 @@ const fs = require('fs');
 const path = require('path');
 const { pathToFileURL } = require('url');
 const { isMainThread } = require('worker_threads');
+const isUnix = process.platform === 'darwin' || process.platform === 'linux';
 
 function rmSync(pathname, useSpawn) {
   if (useSpawn) {
-    const escapedPath = pathname.replaceAll('\\', '\\\\');
-    spawnSync(
-      process.execPath,
-      [
-        '-e',
-        `require("fs").rmSync("${escapedPath}", { maxRetries: 3, recursive: true, force: true });`,
-      ],
-    );
+    if (isUnix) {
+      for (let i = 0; i < 3; i++) {
+        const { status } = spawnSync(`rm -rf ${pathname}`);
+        if (status === 0) {
+          break;
+        }
+      }
+    } else {
+      const escapedPath = pathname.replaceAll('\\', '\\\\');
+      spawnSync(
+        process.execPath,
+        [
+          '-e',
+          `require("fs").rmSync("${escapedPath}", { maxRetries: 3, recursive: true, force: true });`,
```

Comment:
```suggestion
          `fs.rmSync(${JSON.stringify(escapedPath)}, { maxRetries: 3, recursive: true, force: true });`,
```

Reply:
TIL. How come `fs` be global? Is this specific to REPL?

Reply:
Well no, as you can see it also applies to `--eval` :)

Reply:
How come?

```
❯ node
Welcome to Node.js v22.13.0.
Type ".help" for more information.
> typeof fs
'object'
> fs.read
[Function: read]
```

Reply:
It's not clear what you're asking. `fs` and most other node.js built-ins are exposed as globals in the repl and when using `--eval` but not at any other time. In scripts you always need to require/import them.

Reply:
Yes. Today I learned.